src/System.Memory/tests/ReadOnlySpan/ToUpperLower.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+using System.Collections.Generic;
+
+namespace System.Memory.Tests.ReadOnlySpan
```

Comment:
nit: most of the other files here have namespace `System.SpanTests` Might be better to remain consistent

Reply:
ok